plugins/jira-dashboard-backend/src/service/router.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  CacheManager,
+  TokenManager,
+  errorHandler,
+} from '@backstage/backend-common';
+import express from 'express';
+import Router from 'express-promise-router';
+import { Config } from '@backstage/config';
+import { Logger } from 'winston';
+import { CatalogClient } from '@backstage/catalog-client';
+import { DiscoveryApi } from '@backstage/plugin-permission-common';
+import { IdentityApi } from '@backstage/plugin-auth-node';
+import {
+  getFiltersFromAnnotations,
+  getIssuesFromComponents,
+  getIssuesFromFilters,
+  getProjectResponse,
+} from './service';
+import { getDefaultFilters } from '../filters';
+import { Filter, Project, JiraResponse } from '../types';
+import { getProjectAvatar } from '../api';
+import stream from 'stream';
+import {
+  COMPONENT_ANNOTATION,
+  FILTER_ANNOTATION,
+  PROJECT_KEY_ANNOTATION,
+} from '../annotations';
+
+/**
+ * @public
+ */
+export interface RouterOptions {
+  logger: Logger;
+  config: Config;
+  discovery: DiscoveryApi;
+  identity: IdentityApi;
+  tokenManager: TokenManager;
+}
+
+const DEFAULT_TTL = 1000 * 60;
```

Comment:
Should this be configurable? Follow up PR would be fine for this just to keep things rolling 👍

Reply:
Good idea! Next PR it is :)